aws/components/s3/setup.ftl

```diff
@@ -162,6 +162,56 @@
                                                         publicIPWhiteList)]
                         [#break]
                 [/#switch]
+
+                [#local occurrenceNetwork = getOccurrenceNetwork(occurrence) ]
+                [#local networkLink = occurrenceNetwork.Link!{} ]
+                [#if networkLink?has_content ]
```

Comment:
A comment explaining why it is necessary to add vpc access separate from the section that proceeds this would be useful I think for maintainers.

Reply:
I think it might be better to make this an extension, now that multiple extensions can be applied to a bucket this would suit that pattern and remove the dependency on S3 to know about the network